.github/workflows/behat.yaml

```diff
@@ -0,0 +1,167 @@
+name: Behat
+
+on:
+    push:
+        branches: [ master ]
+        paths-ignore:
+            - README.md
+    pull_request:
+        branches: [ master ]
+        paths-ignore:
+            - README.md
+
+jobs:
+    sylius:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [7.4]
+                sylius: ['~1.7.0']
+
+        env:
+            APP_ENV: test
+
+        steps:
+            - name: PHP - Switch
+              run: sudo update-alternatives --set php /usr/bin/php${{ matrix.php }} && php -v
+
+            - name: PHP - Setup timezone
+              run: |
+                  echo "date.timezone=UTC" >> /tmp/timezone.ini; \
+                  sudo mv /tmp/timezone.ini /etc/php/${{ matrix.php }}/cli/conf.d/timezone.ini
+
+            - uses: actions/checkout@v2
+              with:
+                  path: theme
+
+            - uses: actions/cache@v1
+              id: cache-composer
+              with:
+                  path: /home/runner/.composer/cache
+                  key: composer-php.${{ matrix.php }}-sylius.${{ matrix.sylius }}-${{ github.sha }}
+                  restore-keys: composer-php.${{ matrix.php }}-sylius.${{ matrix.sylius }}-
+
+            - name: Composer - Create cache directory
+              run: mkdir -p /home/runner/.composer/cache
+              if: steps.cache-composer.outputs.cache-hit != 'true'
+
+            - name: Composer - Self Update
+              run: |
+                  mkdir -p /home/runner/.composer/
+                  sudo composer self-update
+
+            - name: Composer - Github Auth
+              run: composer config -g github-oauth.github.com ${{ github.token }}
+
+            - name: Composer - Install Sylius-Standard
```

Comment:
There are a lot of similarities with what's happening in the Dockerfile of https://github.com/Sylius/BootstrapTheme/pull/66 here.